templates/heroSlide.hbs

```diff
@@ -0,0 +1,14 @@
+<div class="hero__slide__container {{isDarkTheme}}">
```

Comment:
Using `{{isDarkTheme}}` is creating 2 classes, `true` or `false`. those class names are difficult to understand. You should use a conditional like this `{{#if isDarkTheme}}your-class-name-for-dark-theme{{else}}{{/if}}`. You should only add the dark-theme class when needed, the light one should be your default without needing a class.